cypress/support/commands/commands.js

```diff
@@ -203,6 +203,70 @@ Cypress.Commands.add(
     }
 );
 
+/**
+ * Types in a sw-multi-select field all the specified values and checks if the content was correctly set.
+ * @memberOf Cypress.Chainable#
+ * @name typeMultiSelectAndCheckMultiple
+ * @function
+ * @param {String[]} values - Desired values of the element
+ */
+Cypress.Commands.add(
+    "typeMultiSelectAndCheckMultiple",
```

Comment:
Did you test the command in terms of flakyness? Our `sw-select` is quite prone to such problems, unfortunately.

Reply:
If you can define "flakyness" for me i can test for that :smile: .

I have only tested it on one `sw-multi-select` for now, so I will test it on some others and report back soon.
For the most part I think this command should be more reliable than the other `typeMultiSelectAndCheck` command, because it doesn't depend on hard positions / order of elements and is inspired by that command.

Reply:
Regarding reliabilty:

I replaced 3 occurrences of the old `typeMultiSelectAndCheck` command with the new `typeMultiSelectAndCheckMultiple` command (passed a array with single value as arguments) and it worked every time. I use this command with multiple values as a argument in my current test case and that also has not caused any issues yet.

Reply:
The term "flakyness" can be used as reliabilty. Previously the command failed quite a lot with a false positive (we discussed that already in our private Slack conversation) and the corresponding component has quite a few states which we need to be aware of.

Usually when we're changing commands, we're using the branch of the e2e testsuite and creating an MR and let the pipeline pass 5 times before merging. @leichteckig correct me if I'm wrong here, please.

Reply:
Remember that this MR is not changing any existing commands and only adds a new one (because it comes for different use cases and I don't want to break any existing command / the API of the command).
From my side the command should be reliable enough as far as I stated above.

@leichteckig can we close this conversation or do you want me to do something concrete for this?

Reply:
Can be closed 👍